perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -1112,6 +1114,10 @@ void MapBasedPredictionNode::objectsCallback(const TrackedObjects::ConstSharedPt
 
   // Publish Results
   pub_objects_->publish(output);
+
+  // Publish published time only if there are subscribers more than 1
+  published_time_publisher_->publish(pub_objects_, output.header.stamp);
```

Comment:
We should have named this method as `publish_if_subscribed()`

Could you refactor this function name?
No need to open a new PR, you can do it within this PR.

Reply:
Hi, I agree with you. I changed the function name. 

Also PR for adding unit test was updated: ```publish() -> publish_if_subscribed()```
- https://github.com/autowarefoundation/autoware.universe/pull/6610